src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -1715,7 +1728,13 @@ export default class EthereumApi implements Api {
    * ```
    */
   @assertArgLength(1)
-  async eth_getTransactionByHash(transactionHash: DATA) {
+  async eth_getTransactionByHash(
+    transactionHash: DATA
+  ): Promise<
+    | Ethereum.SignedTransaction<"internal">
+    | Ethereum.PooledTransaction<"internal">
```

Comment:
I find it interesting that we return `"internal"` types from RPC methods. Is that name wrong if we're having to externalized them to users?